website/static/js/requestAccess.js

```diff
@@ -48,7 +48,7 @@ var RequestAccessViewModel = function(currentUserRequestState, nodeId, user) {
             'POST',
             self.updateUrl,
             {
-                'is_cors': true,
+                'isCors': true,
```

Comment:
We have APIv2 tests that active `enforce_csrf` switches enforce csrf and vice versa.  We have APIv2 request access tests. I don't think an additional test is required here - we simply made a typo in passing in the isCors variable in the dict.